tests/xharness/DummyTestProcess/Program.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading;
+using Mono.Options;
+
+namespace DummyTestProcess {
+
+	// The following is a dummy console application that we are going to be using
+	// for testing purposes. The application takes a number of commands that will
+	// help to test diff situations:
+	//
+	// * Exit codes.
+	// * Timeouts
+	// * Child processes.
+	// * Provided parameters
+	//
+	// The console worjs as following:
+	//
+	// -exit-code: Provide the exit code that will be returned after execution
+	// -timeout: The time the application will be idle. This will be used to
+	//			 fake a timeout in the xharness ide.
+	// -children: Create a number of childre processes.
+	//
+	// By default, the application will alsways print the arguments that have been
+	// provided and will run for the given timeout time.
+	class MainClass {
+
+		public static void Main (string [] args)
+		{
+			var timeout = new TimeSpan (0, 1, 0);
```

Comment:
This is more readable:

```suggestion
			var timeout = TimeSpan.FromMinutes(1);
```